internal/net/udp/conn.go

```diff
@@ -74,20 +89,35 @@ func (l *Listener) Close() error {
 }
 
 // Addr returns the listener's network address.
-func (l *Listener) Addr() net.Addr {
+func (l *listener) Addr() net.Addr {
 	return l.pConn.LocalAddr()
 }
 
-// Listen creates a new listener
-func Listen(network string, laddr *net.UDPAddr) (*Listener, error) {
+// ListenConfig stores options for listening to an address.
+type ListenConfig struct {
```

Comment:
This config is not accessible at now. I'd like to expose `internal/net/udp` by moving it to `pion/transport` package.
Then we'll be able to use custom inner listener using https://github.com/pion/dtls/pull/226/files#diff-804732da4aad8a84ebedfb545342f187R26.